### DISCUSSION TITLE
空白页问题

### DISCUSSION THREAD

**Discussion by WeiXikai** [sjtug/SJTUThesis#670] [提问] [answered]
**问题描述**:
我是本科生，在写毕业论文，英文。现在遇到的问题是，有部分章节结束后出现一整个没有页眉页脚的空白页，不知道该如何处理。谢谢。

**系统信息**:

- OS版本(带版本号): Linux
- TeX 发行版(带版本号): TeX Live
**日志信息**:

**Comment by skyzh** [comment, marked as answer]
使用 oneside 模式即可。
